lib/cocoapods-downloader/remote_file.rb

```diff
@@ -173,6 +173,20 @@ def verify_checksum(filename)
           verify_sha1_hash(filename, options[:sha1])
         end
       end
+
+      # @note   The entry is a bundle if it is a directory but has a bundle structure
+      #
+      # @return [Bool] Whether the entry is a bundle
+      #
+
+      def isBundle?(entry)
+        isBundle   = File.extname(entry) == '.framework'
+        isBundle ||= File.extname(entry) == '.app'
+        isBundle ||= File.extname(entry) == '.framework'
```

Comment:
is this duplicated?

Reply:
what about `resource.bundles`?

Reply:
Correct, I accidentally duplicated the line. Will correct and commit change. As for the resource.bundles, should cocoapods check just for the ".bundles" that way it doesn't miss out on non-`resource`?